scio-core/src/main/scala/com/spotify/scio/io/Tap.scala

```diff
@@ -45,6 +47,10 @@ trait Tap[T] { self =>
 
   /** Map items from T to U. */
   def map[U: ClassTag](f: T => U): Tap[U] = new Tap[U] {
+
+    /** Parent Taps this Tap's data came from after [[Tap.map]] */
+    override val parent: Seq[Tap[_]] = self.parent ++ Seq(self)
```

Comment:
It can just be `= self` here. Also unrelated but is `seq :+ self` more idiomatic?